integration/config/core.yaml

```diff
@@ -570,7 +570,7 @@ chaincode:
         # FPC Addition 0: external builder for fpc-c chaincode
         - path: ../../fabric/externalBuilder
           name: fpc-c
-          propagateEnvironmentp:
+          propagateEnvironment:
```

Comment:
this typo was really nasty.

Reply:
it took me an hour to find the reason why ercc does log anything. :/

Reply:
I blame myself for being a bad reviewer :)

Reply:
aargh, sorry for the typo. It worked in the past but i had to rename it due to fabric 2.2 having renamed the vars in a more PC manner and i must have accidentally added a p (annoyingly, it was only in the actually used one but not in commented out version, so most have been added post the global copy/replace ...). At least i did it consistently across the two core.yamls ;-)